tests/bootstrap.php

```diff
@@ -29,6 +30,11 @@
     $fs->remove(__DIR__.'/Fixture/Migrations');
     $fs->mkdir(__DIR__.'/Fixture/Migrations');
 
+    // restore custom migrations
```

Comment:
Can we use `Symfony\Component\Filesystem\Filesystem` to just copy the entire directory?

Reply:
Filesystem only allows to copy files, not directories

Reply:
`Filesystem::mirror()` should do the trick